src/windows.jl

```diff
@@ -176,6 +177,7 @@ function hanning(n::Integer; padding::Integer=0, zerophase::Bool=false)
         0.5*(1+cos(2pi*x))
     end
 end
+hann = hanning
```

Comment:
Shouldn't it be a const?
```suggestion
const hann = hanning
```